satellite/peer.go

```diff
@@ -134,11 +138,12 @@ type Peer struct {
 		Inspector *statdb.Inspector
 	}
 
-	Metainfo struct {
+	Pointerdb struct {
```

Comment:
Let's not change these for now, we'll clean up these later instead.

Reply:
done